drv/lpc55-update-server/src/main.rs

```diff
@@ -69,6 +69,7 @@ const BLOCK_SIZE_BYTES: usize = BYTES_PER_FLASH_PAGE;
 const MAX_LEASE: usize = 1024;
 const HEADER_BLOCK: usize = 0;
 
+const CMPA_FLASH_WORD: u32 = 0x9E40;
 const CFPA_PING_FLASH_WORD: u32 = 0x9E00;
 const CFPA_PONG_FLASH_WORD: u32 = 0x9E20;
 const CFPA_SCRATCH_FLASH_WORD: u32 = 0x9DE0;
```

Comment:
I should probably just bite the bullet and fix the update server to just act on page addresses instead of words given I had a bug in development based on this confusion.

Reply:
I tried to do this and it caused more confusion. Needs more thought so punting on this for now.